README.md

```diff
@@ -111,7 +111,7 @@ Installs the mixlib-install/mixlib-install gems and upgrades the chef-client.
 - `exec_args` - An array of arguments to exec the chef-client with. default: ARGV
 - `download_url_override` - The direct URL for the chef-client package.
 - `checksum` - The SHA-256 checksum of the chef-client package from the direct URL.
-- `upgrade_delay` - The delay in seconds before the scheduled task to upgrade chef-client runs on windows. default: 30
+- `upgrade_delay` - The delay in seconds before the scheduled task to upgrade chef-client runs on windows. default: 61. Lowering this limit is not recommended, there is a risk to schedule upgrade within the same minute that Chef runs and upgrade won't be triggered.
```

Comment:
still think this is very verbose and should just leave it as "(lowering this limit is not recommended)" and throw a more verbose comment in the code.

also "If the upgrade is scheduled in the same minute that Chef runs, there is a risk the upgrade will not be triggered."

Reply:
Moved this stuff to resource description.